### ISSUE TITLE
Improve onnx.ai documentation rendering to include block separators

### ISSUE THREAD

**Issue by robertknight** [onnx/onnx#5872]
The documentation for some attributes is harder to read in the HTML version on onnx.ai compared to the version rendered by GitHub's markdown rendering, due to block/paragraph separators not being preserved.

With the `Resize` operator for example, the meaning of the different values for the `coordinate_transformation_mode` attribute enum are easy to follow [on GitHub](https://github.com/onnx/onnx/blob/main/docs/Operators.md#resize), but hard to follow on https://onnx.ai/onnx/operators/onnx__Resize.html. The description of each enum value is rendered as a separate block on GitHub, but on onnx.ai everything is squashed into one paragraph.

See also an earlier issue about the same attribute in https://github.com/onnx/onnx/issues/4725. The specific complaint in that bug report, about code being rendered as italics, was resolved.

**GitHub rendering:**

Note each enum value is a separate block, so it is easy to see which value goes with which equation.

<img width="1024" alt="github-rendering" src="https://github.com/onnx/onnx/assets/2458/c7431229-4424-4cfd-b522-987d6eca06f2">

**onnx.ai rendering:**

<img width="450" alt="onnx-ai-rendering" src="https://github.com/onnx/onnx/assets/2458/3e8c41ff-6506-4901-9626-cb1167f7e7a9">

**Comment by justinchuby** [comment]
Thanks for reporting this issue! We may need to look into how sphinx is processing the code blocks. In the meantime contributions are welcome if you have a solution in mind.

**Comment by justinchuby** [comment]
https://github.com/onnx/onnx/pull/5873

**Comment by robertknight** [comment]
Thanks for the quick fix!
